doc/content/getting_started/installation.md

```diff
@@ -6,7 +6,8 @@
 In order to install TMAP8, the MOOSE developer environment must be installed. The
 installation procedure depends on your operating system, so click on the MOOSE
 website link below that corresponds to your operation system/platform and follow
```

Comment:
```suggestion
website link below that corresponds to your operating system/platform and follow
```

This was not fixed from my previous review, but I figured out how to suggest it this time. :)